trino/trino.go

```diff
@@ -1043,6 +1115,30 @@ func (qr *driverRows) initColumns(qresp *queryResponse) error {
 	return nil
 }
 
+func (qr *driverRows) updateCallback(id string, stats stmtStats) {
+	if qr.stmt.conn.callback != nil {
```

Comment:
Reverse the condition and do an early return. This allows to avoid one level of indentation, making the whole function more readable

Reply:
Wow... I have never thought of indentation level so at first I did not understand properly on what you meant by reversing the condition. I am grateful for spotting this out.

Reply:
This explains it better: https://go.dev/doc/effective_go#if